webrtc.html

```diff
@@ -2073,40 +2094,45 @@
           <dt>new</dt>
 
           <dd>The <a>ICE Agent</a> is gathering addresses and/or waiting for remote
-          candidates to be supplied.</dd>
+          candidates to be supplied, and has not yet started checking.</dd>
 
           <dt>checking</dt>
 
-          <dd>The <a>ICE Agent</a> has received remote candidates on at least one
-          component, and is checking candidate pairs but has not yet found a
-          connection. In addition to checking, it may also still be
-          gathering.</dd>
+          <dd>The <a>ICE Agent</a> has received a remote candidate for at least
+          one component, and is checking candidate pairs but has not yet found a
+          connection. In addition to checking, it may also still be gathering.</dd>
 
           <dt>connected</dt>
 
           <dd>The <a>ICE Agent</a> has found a usable connection for all components
           but is still checking other candidate pairs to see if there is a
-          better connection. It may also still be gathering.</dd>
+          better connection. It may also still be gathering and/or waiting for
+          additional remote candidates.</dd>
 
           <dt>completed</dt>
 
-          <dd>The <a>ICE Agent</a> has finished gathering and checking and found a
-            connection for all components.
-            Details on how the completed state in ICE is reached are covered in
-            [[!ICE]].  </dd>
+          <dd>The <a>ICE Agent</a> has finished gathering, received an
+          indication that there are no more remote candidates, and finished
+          checking all candidate pairs and found a connection for all
+          components.</dd>
 
           <dt>failed</dt>
 
-          <dd>The <a>ICE Agent</a> is finished checking all candidate pairs and failed
-          to find a connection for at least one component. Connections may have
-          been found for some components.</dd>
+          <dd>The <a>ICE Agent</a> has finished gathering, received an
+          indication that there are no more remote candidates, and finished
+          checking all candidate pairs and failed to find a connection for at
+          least one component. Connections may have been found for some
+          components.</dd>
 
           <dt>disconnected</dt>
 
           <dd>Liveness checks have failed for one or more components. This is
           more aggressive than <code>failed</code>, and may trigger
           intermittently (and resolve itself without action) on a flaky
-          network.</dd>
+          network. Alternatively, the <a>ICE Agent</a> has finished checking
+          all existing candidates pairs and failed to find a connection for
+          at least one component, but is still gathering and/or waiting for
+          additional remote candidates.</dd>
 
```

Comment:
Does this mean that when we have more than one component, we will go new -> disconnected -> completed or new -> disconnected -> failed?